plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -43,22 +43,21 @@
       </libraryFile>
     </library>
     
+    <!-- TODO unify objectify and objectify6 -->
     <library
           id="objectify"
           group="appengine"
           name="Objectify"
           tooltip="%objectify.tooltip"
           dependencies="include"
           siteUri="https://github.com/objectify/objectify/wiki" >
-      <libraryDependency id="appengine-api" />
       <libraryFile
-          pinned="true"
           export="true"
-          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/5.1.22">
+          javadocUri="https://www.javadoc.io/doc/com.googlecode.objectify/objectify/6.0">
         <mavenCoordinates
               artifactId="objectify"
               groupId="com.googlecode.objectify"
-              version="5.1.22" />
+              version="6.0.2" />
```

Comment:
Actually, it may be better to keep Objectify 5, so that existing Java 7 projects can continue working correctly with the Add Library... functionality. Likewise, I think we won't be removing the JRE7 facet for some time.